BUILDING.md

```diff
@@ -77,48 +75,48 @@ $ cd grpc/third_party/protobuf
 $ sudo make install   # 'make' should have been run by core grpc
 ```
 
-# Build from Source
+# Clone the repository (including submodules)
 
-For developers who are interested to contribute, the following commands show how to compile the
-gRPC C Core library.
+Before building, you need to clone the gRPC github repository and download submodules containing source code 
+for gRPC's dependencies (that's done by the `submodule` command or `--recursive` flag). The following commands will clone the gRPC
+repository at the latest stable version.
+
+## Unix
 
 ```sh
  $ git clone -b $(curl -L https://grpc.io/release) https://github.com/grpc/grpc
  $ cd grpc
  $ git submodule update --init
- $ make
- $ [sudo] make install
-```
+ ```
 
 ## Windows
 
-There are several ways to build under Windows, of varying complexity depending
-on experience with the tools involved.
+```
+> @rem You can also do just "git clone --recursive -b THE_BRANCH_YOU_WANT https://github.com/grpc/grpc"
+> powershell git clone --recursive -b ((New-Object System.Net.WebClient).DownloadString(\"https://grpc.io/release\").Trim()) https://github.com/grpc/grpc
+> cd grpc
+> @rem To update submodules at later time, run "git submodule update --init"
+```
 
+# Build from source
```

Comment:
I agree this would be useful. For now, I provided some basic info about why we support multiple build systems, we can elaborate on this in a followup PR (please see my general comment about merging this PR and iterating).

Reply:
I think the intro you provided is sufficient 👍 There is a typo though (should be "would work for all...")